sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2998,6 +2998,14 @@ SIREPO.app.directive('simStatusPanel', function(appState) {
                     || 'Simulation ' + $scope.simState.stateAsText() + ': ' + $scope.simState.getFrameCount() + ' animation frames';
             };
             $scope.start = function() {
+                // The available jobRunModes can change. Default to parallel if
+                // the current jobRunMode doesn't exist
+                if (
+                    appState.models[$scope.simState.model].jobRunMode
+                        in authState.jobRunModeMap === false
+                ) {
+                    appState.models[$scope.simState.model].jobRunMode = 'parallel';
```

Comment:
There is a default value, which we could use. It's very hard to extract values from the schema. `SIREPO.INFO_INDEX_DEFAULT_VALUE = 2` is in sirepo-components, but you have to find the schema value, which is tricky. There should be "defaultValue(model, field)". Ugh. Just noting. Leave the copy-and-paste as it is.